tests/unit/pages/production-schedule.spec.js

```diff
@@ -145,22 +142,25 @@ describe('ProductionSchedule', () => {
     test('onBodyScroll', () => {
     })
 
-    test('convertScheduleItems', () => {
-      let item = wrapper.vm.convertScheduleItems([{
+    test('convertScheduleItems', done => {
+      const item = wrapper.vm.convertScheduleItems(undefined, [{
```

Comment:
Not sure about the `undefined` (that refers to `taskTypeElement`) but that's the only way I found to make it work. Any thought ?

Reply:
This probably requires to add tests to test both cases. But that's not the matter of the PR. So let's merge this.